frontend/src/main/components/RecommendationRequest/RecommendationRequestTable.js

```diff
@@ -16,6 +16,11 @@ export default function RecommendationRequestTable({ requests, currentUser }) {
     navigate(`/requests/edit/${cell.row.values.id}`);
   };
 
+  const formattedDate = (val) => {
```

Comment:
This is much better!

But moving it (and the tests) into a file under `utils` would be even better; because then this can be reused across multiple components.

I'll merge this for now, but I'd encourage you to do a separate PR (for 5 points) where you refactor it further by splitting it out into a separate dateUtils.js file.